k3s.yaml

```diff
@@ -64,37 +64,13 @@ pipeline:
       ./scripts/download
 
       # CVE-2023-47108
-      go mod edit -dropreplace=go.opentelemetry.io/otel/exporters/otlp/otlpmetric
-      go mod edit -dropreplace=go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc
-      go mod edit -dropreplace=go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp
-      go mod edit -dropreplace=go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful
       go mod edit -dropreplace=go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc
-      go mod edit -dropreplace=go.opentelemetry.io/otel
-      go mod edit -dropreplace=go.opentelemetry.io/otel/sdk
-      go mod edit -dropreplace=go.opentelemetry.io/otel/trace
-      go mod edit -dropreplace=go.opentelemetry.io/proto/otlp
-      go mod edit -dropreplace=go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc
-      go mod edit -dropreplace=go.opentelemetry.io/otel/exporters/otlp/otlptrace
-      go mod edit -dropreplace=go.opentelemetry.io/otel/metric
-      go mod edit -dropreplace=go.opentelemetry.io/otel/exporters/otlp/internal/retry
-
-      go get go.opentelemetry.io/contrib/instrumentation/github.com/emicklei/go-restful/otelrestful@v0.46.1
-      go get go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.46.1
-      go get go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.46.1
-      go get go.opentelemetry.io/otel@v1.21.0
-      go get go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc@v0.44.0
-      go get go.opentelemetry.io/otel/exporters/otlp/otlptrace@v1.21.0
-      go get go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc@v1.21.0
-      go get go.opentelemetry.io/otel/metric@v1.21.0
-      go get go.opentelemetry.io/otel/sdk@v1.21.0
-      go get go.opentelemetry.io/otel/trace@v1.21.0
-      go get go.opentelemetry.io/proto/otlp@v1.0.0
+      go get go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.54.0
```

Comment:
Can't we use the go/bump for these as well or is it resulting in an error?

Reply:
@debasishbsws Nice suggestion. Yes this is possible. See https://github.com/wolfi-dev/os/pull/28083 PR to transition to gobump only